src/graphql/mutations.ts

```diff
@@ -1,17 +1,21 @@
+import { getInput } from '@actions/core';
+
+const MERGE_METHOD = getInput('MERGE_METHOD');
```

Comment:
Only input I have would be to validate the input against a map of supported values.